dspace/config/submission-forms.dtd

```diff
@@ -55,3 +55,9 @@
  <!ATTLIST language value-pairs-name CDATA  #IMPLIED>  
 
  <!ELEMENT readonly (#PCDATA) >
+
+<!ELEMENT relationship-type (#PCDATA) >
+<!ELEMENT search-configuration (#PCDATA) >
+<!ELEMENT filter (#PCDATA) >
+<!ELEMENT relation-field (relationship-type, search-configuration, filter?, repeatable?, label, hint, linked-metadata-field?, required?)>
+<!ELEMENT linked-metadata-field (dc-schema, dc-element, dc-qualifier?, input-type)>
```

Comment:
If some of the elements of the `field` element are unsupported for `relation-field`, maybe they are supported here for the `linked-metadata-field` (f.e. `language`)?

Reply:
Because I don't even know what this will do for regular metadata, I'd prefer not to include elements which may make no sense. This can be introduced later when the submission has been finalized

Reply:
`language` will add a select box next to the metadata field to set the `text_lang` value of the metadata value. I think it is already supported by "normal" metadata-only fields.

`style` maybe has something to do with the rendering of the field, but I do not know for sure.

Reply:
while we may need to add additional elements here eventually (like `language`), I'm fine with leaving this as-is for the first commit.  Additional elements can be added once needed.